manifests/init.pp

```diff
@@ -57,26 +57,36 @@
       SOFT_UID_DOMAIN = TRUE
       | EOF
     ,
-    require => Yumrepo['condor'],
+    require => Package['condor'],
   }
 
   file { '/etc/condor/config.d/schedd':
     ensure  => file,
-    content => 'DAEMON_LIST = MASTER, SCHEDD',
-    require => Yumrepo['condor'],
+    content => @(CONTENT)
+      DAEMON_LIST = MASTER, SCHEDD
+      HISTORY_CONTAINS_JOB_ENVIRONMENT = False
+      | CONTENT
+    ,
+    require => Package['condor'],
+  }
+
+  file { '/etc/condor/config.d/dagman':
```

Comment:
The string fragment `/etc/condor/config.d` is repeated multiple times in this file. It should be DRY'd up with a variable.

Reply:
this was DRY'd and moved to config.pp